lib/symboldatabase.cpp

```diff
@@ -768,6 +762,14 @@ void SymbolDatabase::createSymbolDatabaseVariableInfo()
             func->addArguments(this, &scope);
         }
     }
+
+    // fill in variable info. This must be done after arguments so that function declarations already have a populated argument list.
+    // This is required in order to be able to match function calls with declarations so that we don't confuse function calls with
+    // variable declarations.
```

Comment:
I'm not sure about further implications of this change but it doesn't seem to have adversely affected test results